oneview/resource_logical_enclosure.go

```diff
@@ -160,6 +165,7 @@ func resourceLogicalEnclosure() *schema.Resource {
 			},
 			"logical_interconnect_uris": {
 				Optional: true,
```

Comment:
This is a read only field please remove optional from here.

Reply:
done